Formula/lilypond.rb

```diff
@@ -0,0 +1,79 @@
+class Lilypond < Formula
+  desc "Music notation for everyone"
+  homepage "https://lilypond.org"
+  url "https://lilypond.org/download/sources/v2.22/lilypond-2.22.1.tar.gz"
+  sha256 "72ac2d54c310c3141c0b782d4e0bef9002d5519cf46632759b1f03ef6969cc30"
+  license "GPL-3.0-or-later"
+
+  revision 1
+
+  depends_on "autoconf" => :build
+  depends_on "bison" => :build # because macOS version is not supported
+  depends_on "dblatex" => :build
+  depends_on "extractpdfmark" => :build
+  depends_on "fontconfig" => :build
+  depends_on "fontforge" => :build
+  depends_on "freetype" => :build
+  depends_on "gettext" => :build
+  depends_on "ghostscript" => :build
+  depends_on "glib" => :build
+  depends_on "guile@2" => :build
+  depends_on "imagemagick" => :build
+  depends_on "make" => :build
+  depends_on "pango" => :build
+  depends_on "pkg-config" => :build
+  depends_on "t1utils" => :build
+  depends_on "texi2html" => :build
+  depends_on "texinfo" => :build # because macOS version is not supported
+
+  depends_on "fontconfig"
+  depends_on "freetype"
+  depends_on "ghostscript"
+  depends_on "guile@2"
+  depends_on :macos
+  depends_on "pango"
+  depends_on "python@3.9"
```

Comment:
If formulae are already declared this way as dependencies there's no need to declare them a second time with `=> :build`.